src/eval-frame/components/panes/console/console-input-field.jsx

```diff
@@ -154,7 +156,9 @@ export const ConsoleInputMessagePasser = connectMessagePassers(
       postActionToEditor({
         type: "CONSOLE_HISTORY_MOVE",
         consoleCursorDelta
-      })
+      }),
+    evalConsoleInput: consoleText =>
+      postMessageToEditor("CONSOLE_NEEDS_EVALUATION", consoleText)
```

Comment:
the substantive change in this file is that this component no longer dispatches, but rather sends a message  to the editor to trigger a console eval

Reply:
Is there a reason we're not going through `sendActionToEditor` at this point instead? Seems like it would make it easier to follow along vs. forcing me to look up the diffs b/t `postMessageToEditor` and `sendActionToEditor`.

Reply:
as in the case above, the message sent here is not a redux action that is being sent up, it is it's own message type that causes the message receiver in the editor to dispatch on the editor side.

the fact that this stuff is so confusing indicates the need for more refactor!